web/controllers/wanikani.js

```diff
@@ -0,0 +1,60 @@
+
+const express = require('express')
+const router = express.Router();
+const wk = require('../../services/wanikani.js')
+
+// TODO: refactor this out! /////////////////////
+const db = require("../../database.js")
+function dateString(d) {
+	let mm = d.getMonth() + 1
+	if(mm < 10) { mm = "0" + mm }
+	let dd = d.getDate()
+	if(dd < 10) { dd = "0" + dd }
+	return d.getFullYear() + "-" + mm + "-" + dd
+}
+async function waniGetTokens() {
```

Comment:
it'd be awesome if we could decouple database operations from the core `./app.js`

Reply:
I agree